dev-requirements.txt

```diff
@@ -0,0 +1,11 @@
+pytest
```

Comment:
You will probably need a .readthedocs.yaml file now that you have separated your requirements.

Reply:
I can add this now.

Reply:
thanks!